src/main/java/com/spotinst/sdkjava/model/ElastigroupConverter.java

```diff
@@ -2504,5 +2665,353 @@ private static PredictiveScale toBl(ApiPredictiveScale predictiveScale) {
 
         return retVal;
     }
+
+    public static ElastigroupStartDeployment toBl(ApiElastigroupStartDeployment elastigroupDeployment) {
+        ElastigroupStartDeployment retVal = null;
+
+        if (elastigroupDeployment != null) {
+            retVal = new ElastigroupStartDeployment();
+
+            if (elastigroupDeployment.isBatchSizePercentageSet()) {
+                retVal.setBatchSizePercentage(elastigroupDeployment.getBatchSizePercentage());
+            }
+
+            if (elastigroupDeployment.isDrainingTimeoutSet()) {
+                retVal.setDrainingTimeout(elastigroupDeployment.getDrainingTimeout());
+            }
+
+            if (elastigroupDeployment.isGracePeriodSet()) {
+                retVal.setGracePeriod(elastigroupDeployment.getGracePeriod());
+            }
+
+            if (elastigroupDeployment.isHealthCheckTypeSet()) {
+                retVal.setHealthCheckType(AwsElastigroupHealthCheckTypeEnum.fromName(elastigroupDeployment.getHealthCheckType()));
+            }
+
+            if (elastigroupDeployment.isStrategySet()) {
+                retVal.setStrategy(toBl(elastigroupDeployment.getStrategy()));
+            }
+        }
+
+        return retVal;
+    }
+
+    private static ElastigroupStartDeploymentStrategy toBl(ApiElastigroupStrategy elastigroupStrategy) {
+        ElastigroupStartDeploymentStrategy retVal = null;
+
+        if (elastigroupStrategy != null) {
+            retVal = new ElastigroupStartDeploymentStrategy();
+
+            if (elastigroupStrategy.isActionSet()) {
+                retVal.setAction(AwsElastigroupActionEnum.fromName(elastigroupStrategy.getAction()));
+            }
+
+            if (elastigroupStrategy.isBatchMinHealthyPercentageSet()) {
+                retVal.setBatchMinHealthyPercentage(elastigroupStrategy.getBatchMinHealthyPercentage());
+            }
+
+            if (elastigroupStrategy.isOnFailureSet()) {
+                retVal.setOnFailure(toBl(elastigroupStrategy.getOnFailure()));
+            }
+
+        }
+
+        return retVal;
+    }
+
+    private static ElastigroupStartDeploymentOnFailure toBl(ApiElastigroupStartDeploymentOnFailure elastigroupOnFailure) {
```

Comment:
rename input parameter to strategyOnFailure

Reply:
Done